discussion/app/views/fragments/comment.scala.html

```diff
@@ -43,7 +43,7 @@
                     </a>
                 }
                 <div class="d-comment__timestamp">
-                    <a href="@Configuration.discussion.url/comment-permalink/@comment.id" class="d-comment__timestamp block-time">
+                    <a href="@Configuration.discussion.url/comment-permalink/@comment.id" class="d-comment__timestamp block-time" data-link-name="Comment permalink">
```

Comment:
@rich-nguyen Is it possible to track this, as it is an outbound request, the tracking call fails to complete as they are leaving the site